builds/jenkins-build-pack-function/Jenkinsfile

```diff
@@ -1268,8 +1307,8 @@ def isExistingLogGroup (groupName, credsId){
 	def filterLogsJSON = toJSON.parseText(filterLogs)
 	def logGroups = filterLogsJSON.logGroups
 	if(logGroups.size() > 0  && logGroups[0].logGroupName == groupName){
-		isExistingLogGrp = true
-	}
+		isExistingLogGrp = trues
```

Comment:
typo: `trues`?

Reply:
My bad. I think the save- ctrl+s caused the issue. Fixed